src/NServiceBus.AcceptanceTests/Routing/When_using_instance_ids.cs

```diff
@@ -37,20 +38,31 @@ public class UnawareSender : EndpointConfigurationBuilder
         {
             public UnawareSender()
             {
-                EndpointSetup<DefaultServer>(c => { c.Routing().RouteToEndpoint(typeof(MyMessage), GetReceiverEndpoint()); });
+                EndpointSetup<DefaultServer>((c,r) =>
+                {
+                    c.UseTransport(r.GetTransportType()).Routing().RouteTo(typeof(MyMessage), GetReceiverEndpoint());
+                });
             }
         }
 
         public class AwareSender : EndpointConfigurationBuilder
         {
             public AwareSender()
             {
-                EndpointSetup<DefaultServer>(c =>
+                EndpointSetup<DefaultServer>((c, r) =>
                 {
-                    c.Routing().RouteToEndpoint(typeof(MyMessage), GetReceiverEndpoint());
-                    c.Routing().Mapping.Physical.Add(new EndpointInstance(GetReceiverEndpoint(), "XYZ"));
+                    c.UseTransport(r.GetTransportType()).Routing().RouteTo(typeof(MyMessage), GetReceiverEndpoint());
+                    c.EnableFeature<SpecificRoutingFeature>();
                 });
             }
+
+            class SpecificRoutingFeature : Feature
```

Comment:
I'm perfectly fine with the scenario. But I wonder whether it would be easier to use `config.Settings.GetOrCreate<EndpointInstances>().Add(...)` directly instead of introducing a feature and making this config available for them?